docs/src/main/asciidoc/mp/integrations/oci.adoc

```diff
@@ -29,80 +29,233 @@ include::{rootdir}/includes/mp.adoc[]
 
 - <<Overview, Overview>>
 - <<maven-coordinates, Maven Coordinates>>
-- <<Usage, Usage>>
-- <<Examples, Examples>>
+- <<_accessing_oci_services, Accessing OCI Services>>
 - <<References, References>>
 
 == Overview
 
-Helidon MP OCI Integration provides easy access to Oracle Cloud Infrastructure using the OCI Java SDK.
+You can quickly and easily deploy Helidon MP applications on Oracle Cloud Infrastructure (OCI) and integrate them with OCI services using the OCI Java SDK and the Helidon OCI SDK Extension.
```

Comment:
Just Helidon applications. There's no need to distinguish between MP and SE here